projects/dateparser/project.yaml

```diff
@@ -0,0 +1,11 @@
+homepage: "https://dateparser.readthedocs.io/en/latest/"
+language: python
+primary_contact: "opensource@zyte.com"
+auto_ccs: 
+  - "ennamarie19@gmail.com"
+fuzzing_engines: 
+  - libfuzzer
+sanitizers: 
+  - address
+  - undefined
```

Comment:
Does the code rely on native code compilation? If not, then we only need one sanitizer (just make it address). Could you confirm?

Reply:
I don't believe it does. Should be fixed now!